src/Services/Air/templates/AIR_AVAILABILTIY_REQUEST.handlebars.js

```diff
@@ -11,9 +11,7 @@ module.exports = `
                 <com:OverridePCC ProviderCode="{{provider}}" PseudoCityCode="{{emulatePcc}}"/>
```

Comment:
Shouldn't it also have `../provider` updated?

Reply:
No, as this part uses `if` statement instead of going into context. Tried the same with `nextResultReference`, but it didn't work